Documentation/postmortems/v3.5-data-inconsistency.md

```diff
@@ -0,0 +1,138 @@
+# v3.5 data inconsistency postmortem
+
+|         |            |
+|---------|------------|
+| Authors | serathius@ |
+| Date    | 2022-04-20 |
+| Status  | draft      |
+
+## Summary
+
+|         |                                                                                                                                                                                                                               |
+|---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Summary | Code refactor resulted in consistent index not being saved atomically, allowing for crash causing inconsistency with other cluster members.                                                                                   |
```

Comment:
"allowing for crash causing..." ->  Independent crash could lead to an inconsistency with other cluster members or a committed transaction being not reflected in the persisted state.